modules/core/shared/src/main/scala/lucuma/core/model/SourceProfile.scala

```diff
@@ -20,20 +20,58 @@ import shapeless.tag.@@
 
 import scala.collection.immutable.SortedMap
 
-sealed trait SourceProfile extends Product with Serializable
+sealed trait SourceProfile extends Product with Serializable {
+
+  /**
+   * Convert to a `Point` source profile. Units are converted to their `Integrated` versions, if
+   * necessary.
+   */
+  def toPoint: SourceProfile.Point
+
+  /**
+   * Convert to a `Uniform` source profile. Units are converted to their `Surface` versions, if
```

Comment:
I guess they are converted without changing the number right? only the units

Reply:
Yes, only the units.